app/views/audits/common/_title.html.erb

```diff
@@ -0,0 +1,4 @@
+<div class="form-group">
+  <%= label_tag 'query', 'Search term:', class: '' %>
```

Comment:
I would probably make this `Search in title...` as otherwise it might be misunderstood that it searches across all text (or maybe check with content designer).

Reply:
I just reused the same label as in the CPM that was user researched. I would prefer to stick to existing patterns for the UI

Reply:
CPM is a slightly different context though. For example it doesn't ever display the summary for a piece of content, whereas the Audit Tool does. So for CPM `text` is always title but here we have two fields with text from a content item. I think it's worth asking a third party what they think. Or we can leave this until the next round of user research... It's only a label.

Reply:
Yeah, I would prefer to stick to what we have. Worth discussing with Joe, though.